organisms/management/commands/organisms_create_or_update.py

```diff
@@ -25,19 +24,26 @@
 
 
 class Command(BaseCommand):
-    # "option_list" sets up the command-line options so that the user can enter
-    # the organism attributes.
-    option_list = BaseCommand.option_list + (
-        make_option(
-            '--taxonomy_id', action='store', dest='taxonomy_id',
-            help="Organism Taxonomy ID supplied by NCBI"),
-        make_option(
-            '--common_name', action='store', dest='common_name',
-            help="Organism common name, e.g. 'Human'"),
-        make_option(
-            '--scientific_name', action='store', dest='scientific_name',
-            help="Organism scientific/binomial name, e.g. 'Homo sapiens'"),
-    )
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--taxonomy_id',
+            type=int,
+            dest='taxonomy_id'
```

Comment:
Checking that you're aware dest is redundant here.

Reply:
Thanks. I do want to specify `dest` explicitly here.